### ISSUE TITLE
Import feature

### ISSUE THREAD

**Issue by jcalcada** [PagerDuty/pagerduty-bulk-user-mgr-sample#6]
Hi Scott,  the import feature seems to not work. Is that by design?

**Comment by stmcallister** [comment]
No. 😬 Sorry about that. I'll put this on my list of things to look into. Can you tell me what kind of behavior you're seeing?

**Comment by IvanRodrigues-PD** [comment]
Hi @stmcallister. I tried the import and had the same issue. There is a customer I'm talking to that is very much interested in this functionality. Do you have any idea of when you might be able to take a look into this? 

Thanks.

**Comment by stmcallister** [comment]
Hi guys! I've been updating this project to support the new PDJS library and I'm able to run a successful import. Since this is a sample project, the error messaging isn't awesome. My guess is that your columns in the CSV file aren't named correctly. This is the guidance we give on the import page

```
Required column headers: name, email; optional column headers: time_zone, color, role, job_title, avatar_url, 
description
```

**Comment by geethpd** [comment]
Hi Scott! Just trying this out myself. I did add the column headers and the user list now appears in the import user page however it appears to endlessly attempt to add the 2 users. Inspecting the page doesn't show any error messages. I do have enough user space and the users were not already existing in the account:
![PagerDuty_User_Management_Tool](https://user-images.githubusercontent.com/40037620/122082332-eb069900-cdcd-11eb-940d-bb6396addb6c.jpg)

**Comment by stmcallister** [comment]
Hey y'all! I was back in this project fixing another issue and was reminded about this one. I still can't repro. I was able to import two users using the following format

```
name,email
Ravi Coltrane,smcallister+ravi@pagerduty.com
Ted Lasso,smcallister+tedlasso@pagerduty.com
```

**Comment by jcalcada** [comment]
Hi Scott, I just tried to use the same format for input file as and although the spinner disappears nearly immediately on investigation the 2 users were not added. Maybe there is a something in the code specific to your oauth/id/user etc.
